SocialCareCaseViewerApi/V1/Gateways/DatabaseGateway.cs

```diff
@@ -167,6 +167,15 @@ public DatabaseGateway(DatabaseContext databaseContext, IProcessDataGateway proc
                     ).AsNoTracking().ToList();
             }
 
+            foreach (var allocation in allocations)
+            {
+                if (allocation.AllocatedWorker != null && allocation.CaseStatus?.ToLower() != "closed")
+                {
+                    var teamAllocation = _databaseContext.Allocations.FirstOrDefault(x => x.PersonId == allocation.PersonId && x.WorkerId == null && x.MarkedForDeletion == false);
```

Comment:
What if person has multiple team allocations? Without using team id as well it'll be tricky to fetch the correct record